packages/react-scripts/scripts/eject.js

```diff
@@ -28,8 +31,15 @@ prompt(
 
   console.log('Ejecting...');
 
+  // NOTE: get ownPath and appPath from config/paths.js ?
   var ownPath = path.join(__dirname, '..');
-  var appPath = path.join(ownPath, '..', '..');
+  var appPath;
```

Comment:
Consider reducing this to:

`var appPath = reactScriptsLinked ? path.resolve('.') : path.join(ownPath, '..', '..');`